modules/dragster/src/main/ts/ephox/dragster/datatransfer/Items.ts

```diff
@@ -0,0 +1,37 @@
+import { Type } from '@ephox/katamari';
+
+import { isInReadWriteMode } from './Mode';
```

Comment:
Maybe namespace import this since `Mode.isInReadWriteMode` is a bit more clear that it's a external module function. This module is small now but just a good practice that I've been told that we should follow that I sometimes forget to follow my self. :smile:

Reply:
Good to know, also did it for `DataTransfer` and `DragImage` modules in dragster.